pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -101,6 +105,28 @@ python3.pkgs.buildPythonApplication rec {
     installShellCompletion --bash data/shell-completions/bash/meson
   '';
 
+  # while building documentation meson may want to run binaries for host
+  # which needs an emulator
+  # example of an error which this fixes
+  # [Errno 8] Exec format error: './gdk3-scan'
+  passthru.crossEmulatorWrapper =
```

Comment:
I don't really like mashing things in `passthru`. How do you all feel about putting this in `pkgs/build-support/setup-hooks`?

Reply:
i moved the hook to all-packages.nix but i put the file in the meson dir instead of setup-hooks